diff --git a/install.sh b/install.sh
index 5389c92..35e5e45 100755
--- a/install.sh
+++ b/install.sh
@@ -26,9 +26,4 @@ mv anki.xpm anki.png ${PREFIX}/share/pixmaps/;\
 mv anki.desktop ${PREFIX}/share/applications/;\
 mv anki.1 ${PREFIX}/share/man/man1/)
 
-xdg-mime install anki.xml --novendor
-xdg-mime default anki.desktop application/x-colpkg
-xdg-mime default anki.desktop application/x-apkg
-xdg-mime default anki.desktop application/x-ankiaddon
-
 echo "Install complete. Type 'anki' to run."
diff --git a/uninstall.sh b/uninstall.sh
index 77b71db..8dbbe22 100755
--- a/uninstall.sh
+++ b/uninstall.sh
@@ -6,8 +6,6 @@ if [ "$PREFIX" = "" ]; then
 	PREFIX=/usr/local
 fi
 
-xdg-mime uninstall ${PREFIX}/share/anki/anki.xml || true
-
 rm -rf ${PREFIX}/share/anki
 rm -rf ${PREFIX}/bin/anki
 rm -rf ${PREFIX}/share/pixmaps/anki.xpm
